conftest.py

```diff
@@ -5,6 +5,7 @@
 import tensorflow as tf
```

Comment:
Please reorder all imports according to PEP8 followed by imports from `art` and finally `tests`.